src/Form/_Form.scss

```diff
@@ -115,6 +115,20 @@ $form-control-icon-width: 32px !default;
       padding-right: calc(#{$input-padding-y-sm} - #{2 * $input-border-width});
     }
   }
+
+  .btn-icon {
+    color: $gray-500 !important;
+    &:hover {
+      background-color: $gray-500 !important;
+      color: $white !important;
+    }
+    &:focus {
+      background-color: $white !important;
+      color: $gray-500 !important;
+      -webkit-box-shadow: inset 0 0 0 2px $gray-500 !important;
+      box-shadow: inset 0 0 0 2px $gray-500 !important;
+    }
```

Comment:
Why are there `!important` rules here? We should avoid using !important anywhere in Paragon.